### PR TITLE
Add contributing guide

### LINKED ISSUE #19: Add a contributors guide
Please add guidelines and requirements for contributors.

### PR DESCRIPTION
**Pull request by chrismgonzalez**
This PR adds a 1st draft of the contributing guide.  

Closes #19

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Implementation details are still being discussed and the project is currently at a very early stage.  The stage of the project is changing on a daily basis, thoughts and ideas are being contributed at a very high pace.
+
+## What do we need help with right now?
+
+We are in the very early stages of development on this new application. We value your insight and expertise.  In order to prevent duplicate issues, please search through our existing issues to see if there is one for which you would like to provide feedback. We are currently trying to consolidate many of the issues based on topics (Docs, UI/UX, API architecture, Tech Stack, etc)
+
+## How to submit an issue
+
+- Search through current issues to verify that you are not creating a duplicate issue.
+- Use the issue template as a guide to tell us more about your issue:
```
Comment (ScottBrenner) [inline CONTRIBUTING.md]: Couldn't [GitHub issue templates](https://help.github.com/en/articles/creating-issue-templates-for-your-repository) be used for this?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+  - **Additional context**
+      Add any other context or screenshots about the feature request here.
+
+- Be as descriptive as possible.  At this early stage, it is important to have qualitative evidence to support or dissent on an issue.
+
+## Working on a change
+
+- Create a new branch on your fork: `git checkout -b your-branch-name`
+- Use the Github editor or your local environment to make your changes
```
Comment (ScottBrenner) [inline CONTRIBUTING.md]:
```suggestion
- Use the GitHub editor or your local environment to make your changes
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- Push your changes to the branch you created on your fork: `git push origin your-branch-name`
+
+## How to submit a pull request
+
+- After working on changes and pushing to your fork of the repo, create a pull request on your fork.
+- Complete the PR template:
```
Comment (ScottBrenner) [inline CONTRIBUTING.md]: Again, [GitHub's pull request templates](https://help.github.com/en/articles/creating-a-pull-request-template-for-your-repository) would be useful for this.

**Comment by QuincyLarson** [comment]
@chrismgonzalez This is a great start. Thanks for creating this.

**Comment by chrismgonzalez** [comment]
My pleasure!

**Pull request merged**